website/content/docs/job-specification/disconnect.mdx

```diff
@@ -0,0 +1,199 @@
+---
+layout: docs
+page_title: disconnect Block - Job Specification
+description: |-
+  The "disconnect" block describes the behavior of both the Nomad server and 
+  client in case of a network partition, as well as how to reconcile the workloads
+  in case of a reconnection.
+---
+
+# `disconnect` Block
+
+<Placement groups={['job', 'group', 'disconnect']} />
+
+The `disconnect` block describes the system's behavior in case of a network 
+partition.
```

Comment:
Let's put this right at the very top, so it's clear what this block is for.

```suggestion
The `disconnect` block describes the system's behavior in case of a network 
partition. By default, without a `disconnect` block, if an allocation is on a 
node that misses heartbeats, the allocation will be marked `lost` and will be 
rescheduled.
```